README.md

```diff
@@ -125,14 +125,14 @@ Usage: docker volume create [<VOLUME_NAME>]
 <pre>
 $ docker run \
 --detach \
---publish [HOST_PORT]:[CONTAINER_PORT] \
 --volume 9c-volume:/app/data \
-planetariumhq/ninechronicles-headless \
-<a href = "#run" title="NineChronicles Standalone options">[NineChronicles Standalone Options]</a>
+planetariumhq/ninechronicles-headless:latest \
+<a href = "#run" title="NineChronicles Headless options">[NineChronicles Headless Options]</a>
 </pre>
 #### Note)
-* Instead of including the "--ice-server" option, allocate an Elastic IP to your instance and include it as the "--host" option and after adding an inbound port, include it as the "--port" option. Refer to these official docs on [Elastic IP allocation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html) and [editing inbound rules](https://docs.aws.amazon.com/cli/latest/userguide/cli-services-ec2-sg.html) for more information.
-* For mining, make sure to include "--private-key" option with your private key. Also, include "--libplanet-node" to run the default libplanet node. 
+* If you want to use the same headless options as your game client, refer to **`config.json`** under **`%localappdata%\Programs\Nine Chronicles\resources\app`**.
+* If you are using an [Elastic IP](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html) on your AWS instance, you do not need to include the `--ice-server` option.
```

Comment:
Is it okay not to comment other parameters (i.e., `--host` and `--port`) commented in README before changed?

Reply:
After removing `--host` and `--port` args in Dockerfile(https://github.com/planetarium/NineChronicles.Headless/pull/206/files), users no longer need to include those parameters to run the headless application on AWS so I removed those comments. I suppose I can add the description on `--host` and `--port` if users were to use an Elastic IP.

Reply:
IMO it should comment more before removing it. Libplanet requires either `--ice-server` and `--host`.

```csharp
            if (_host == null && (_iceServers == null || !_iceServers.Any()))
            {
                throw new ArgumentException(
                    $"Swarm requires either {nameof(host)} or " +
                    $"{nameof(iceServers)}."
                );
            }
```

Also the option, `--host`'s default value is `null` so it should comment about the option.